MAUI/DigitalGauge/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: About Maui Digital Gauge control | Syncfusion
+description: Learn here all about introduction of Syncfusion Maui Digital Gauge (SfDigitalGauge) control, its elements and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+# Maui Digital Gauge (SfDigitalGauge) Overview
+
+The digital gauge control is used to display alphanumeric characters in digital (LED display) mode. This control displays a range of values that uses character in combination with numbers.
```

Comment:
The Syncfusion .NET MAUI DigitalGauge control is used to display alphanumeric characters in digital (LED display) mode. This control displays a range of values that use characters in combination with numbers.